moveit_ros/moveit_servo/include/moveit_servo/servo_node.hpp

```diff
@@ -69,6 +69,7 @@ class ServoNode
   // Disable copy assignment.
   ServoNode& operator=(ServoNode&) = delete;
 
+  // NOLINTNEXTLINE(readability-identifier-naming)
   rclcpp::node_interfaces::NodeBaseInterface::SharedPtr get_node_base_interface();
```

Comment:
Rather than suppressing this, can we change it to `getNodeBaseInterface()`? Or is this something specifically inherited from rclcpp that we can't control?